virt/logging_events_monitoring/virt-monitor-vmi-migration.adoc

```diff
@@ -16,3 +16,10 @@ You can monitor the progress of all live migrations on the xref:../../virt/virt-
 You can view the migration metrics of a virtual machine on the xref:../../virt/virt-web-console-overview.adoc#ui-virtualmachine-details-metrics_virt-web-console-overview[*VirtualMachine details -> Metrics* tab] in the web console.
```

Comment:
Not sure why, but this link doesn't always work. More often than not, it General tab table. Maybe just a Netlify thing?